src/app/pages/details/customization-form/module-list.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import {useToggle, RawHTML, useDataFromPromise} from '~/components/jsx-helpers/jsx-helpers.jsx';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import bookToc from '~/models/book-toc';
+import './module-list.css';
+
+function ContentGroup({item, children}) {
+    const [expanded, toggleExpanded] = useToggle();
+    const icon = expanded ? 'caret-down' : 'caret-right';
+
+    return (
+        <React.Fragment>
+            <div
+                className="toggle" onClick={() => toggleExpanded()}
```

Comment:
since `toggleExpanded` isn't a member function that would lose it's context, there's no need to wrap it, you could:
`className="toggle" onClick={toggleExpanded}`